cognite/neat/_session/_wrappers.py

```diff
@@ -11,7 +11,7 @@ class HasStore(Protocol):
     _store: NeatStore
 
 
-T_Class = TypeVar("T_Class", bound=object)
+T_Class = TypeVar("T_Class", bound=HasStore)
```

Comment:
This fixes the assumption of the `session_wrapper`. Adding this cause `MyPy` to catch the following

<img width="1062" height="125" alt="Image" src="https://github.com/user-attachments/assets/579b14dc-95ca-4ed1-9f03-26c098ffe244" />